doc/user-guide/Getting-started.md

```diff
@@ -2,59 +2,74 @@
 
 ### Getting started with MongooseIM on Mac OS X
 
-This short guide will show you how to compile MongooseIM from source code or install binaries from a DMG image on Mac OS X, and get users chatting right away.
+This short guide will show you how to compile MongooseIM from source code or install binaries from a DMG image on macOS, and get users chatting right away.
```

Comment:
It's called "macOS" since Sierra but what about earlier versions? :) Maybe a good compromise would be to simply say "(...)install binaries from a DMG image on Mac, and get(...)"?